indico/htdocs/js/indico/jquery/actioninput.js

```diff
@@ -32,7 +32,7 @@
 
             self._super();
 
-            self.actionIcon = $('<a class="i-link accept {0}"></a>'.format(self.options.actionIcon))
+            self.actionIcon = $('<a class="i-link accept {0} hide-if-locked"></a>'.format(self.options.actionIcon))
```

Comment:
I think it was you mentioning that the static classes go before the dynamic ones :wink: